l10n_fi_payment_terms/data/payment_terms_finnish.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+    
+        <!--
+        	Payment terms
+        -->
+        
+        <!-- Delete the defaults -->
+        <delete model="account.payment.term" id="account.account_payment_term_immediate"/>
```

Comment:
A bit hazardous for non-fresh installations. I'd leave the original payment terms untouched.